packages/javalib.2.3.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.1.1"
+name: "javalib"
+maintainer: "sawja@inria.fr"
+version: "2.3.1"
```

Comment:
Hello David,

According to https://opam.ocaml.org/doc/Packaging.html, "The 
|opam-version|, |maintainer| and |version| fields are mandatory " .

So I'm a bit confused  about the "version" field.

Nevertheless, I removed it. Do you think  the opam files below are 
correct (one for javalib and the following for the one related to the 
sawja project which uses Javalib.)?

If so, i will update them right after.

thanks for your help

Laurent


----------begin ------------

opam-version: "1.1"
maintainer: "sawja@inria.fr"
build: [
   ["./configure.sh"]
   [make "ptrees"]
   [make "installptrees"]
   [make]
]
install: [make install]
remove: [
   ["ocamlfind" "remove" "javalib"]
   ["ocamlfind" "remove" "ptrees"]
]
depends: [
   "ocamlfind"
   "camlzip" {= "1.05"}
   "extlib" {>= "1.5.1"}
]

-----------end--------------

----------begin of opam file for sawja -------------

opam-version: "1.1"
maintainer: "sawja@inria.fr"
build: [
   ["./configure.sh"]
   [make]
]
install: [make "install"]
remove: [
   ["ocamlfind" "remove" "sawja"]
]
depends: [ "ocamlfind" "javalib" {= "2.3.1"} ]

----------end of opam file for sawja






On 07/04/2015 12:16 AM, David Sheets wrote:
>
> In packages/javalib.2.3.1/opam 
> <https://github.com/ocaml/opam-repository/pull/4370#discussion_r33882013>:
>
> > @@ -0,0 +1,22 @@
> > +opam-version: "1.1.1"
> > +name: "javalib"
> > +maintainer: "sawja@inria.fr"
> > +version: "2.3.1"
>
> Could you please remove |name| and |version| when publishing? It helps 
> keep the repository file reusable for variants.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/ocaml/opam-repository/pull/4370/files#r33882013>.
>